Modules/Filtering/DisplacementField/test/itkInverseDisplacementFieldImageFilterTest.cxx

```diff
@@ -161,6 +146,16 @@ itkInverseDisplacementFieldImageFilterTest(int argc, char * argv[])
     ++it;
   }
 
+  // Test non-default values
+  auto spacing = itk::MakeFilled<DisplacementFieldType::SpacingType>(9876.0);
```

Comment:
Another option might be to use some "extreme" values from `std::numeric_limits<T>`.  (Again, this is just a suggestion.)

Reply:
I was looking into the "magic-numbers" clang-tidy, but had not tested that yet.

I did pick 9876 and 1234 as semi-obvious "extreme values".  I agree it would be better to have them named.  I'll do that when I look at the magic-numbers changes.